doc/connectivity/bluetooth/api/mesh/access.rst

```diff
@@ -246,6 +246,14 @@ message, it will send messages with delay close to expiration to free memory.
 When the mesh stack is suspended or reset, messages not yet sent are removed and
 the :c:member:`bt_mesh_send_cb.start` callback is raised with an error code.
 
+.. note::
+   When a model sends several messages in a row, it may happen that the messages are sent not in
+   the order they were passed to the access layer. This is because some messages can be delayed
+   for a longer time than the others.
+
```

Comment:
```suggestion
   When a model sends several messages in a row, it may happen that the messages are not sent in
   the order they were passed to the access layer. This is because some messages can be delayed
   for a longer time than the others.

```